Makefile

```diff
@@ -2,16 +2,8 @@
 SRCS := $(shell find . -type d -name 'vendor' -prune -o -name '*.go' -print)
 
 .PHONY: install
-install: glide #download dependencies (including test deps) for the package
-	glide install
-
-.PHONY: update
-update: glide #updates dependencies used by the package and installs them
-	glide update
-
-.PHONY: glide
-glide: # ensure the glide package tool is installed
-	which glide || go get github.com/Masterminds/glide
```

Comment:
Would modify the test target below to follow the same pattern as here https://github.com/lyft/go-lyft-configset/pull/107/files#diff-b67911656ef5d18c4ae36cb6741b7965L3